packages/shared/src/components/auth/OnboardingSteps/ContentTypes/CustomCheckbox.tsx

```diff
@@ -0,0 +1,47 @@
+// TODO: move in a new component in shared if the onboardingContentType experiment is successful
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { Checkbox } from '../../../fields/Checkbox';
+
+interface CustomCheckboxProps {
+  checked: boolean;
+  title: string;
+  description: string;
+  onCheckboxToggle: () => void;
+  name: string;
+}
+export const CustomCheckbox = ({
+  checked,
+  title,
+  description,
+  onCheckboxToggle,
+  name,
+}: CustomCheckboxProps): ReactElement => {
+  return (
+    <button
+      type="button"
+      className={classNames(
+        'h-[8.25rem] max-w-80 cursor-pointer rounded-16 border p-4 text-left',
+        checked
+          ? 'border-accent-cabbage-bolder bg-surface-float text-text-primary'
```

Comment:
In the design, the border (and checkboxes) use a gradient instead of solid colour. Is this intended to differ?

Reply:
Because this component was created and accepted in a very recent implementation for layout on onboarding, decided to go this way without the gradient. Once this experiment wins will add the gradient for sure, just not to put in extra work now.

Reply:
https://dailydotdev.atlassian.net/browse/MI-451